app/src/main/java/org/mozilla/fenix/components/FenixSnackbar.kt

```diff
@@ -63,6 +75,34 @@ class FenixSnackbar private constructor(
                 it.duration = duration
             }
         }
+
+        // Use the same implementation of `Snackbar`
+        @SuppressWarnings("MagicNumber")
+        private fun findSuitableParent(_view: View?): ViewGroup? {
+            var view = _view
+            var fallback: ViewGroup? = null
+
+            do {
+                if (view is CoordinatorLayout) {
+                    return view
+                }
+
+                if (view is FrameLayout) {
+                    if (view.id == 16908290) {
```

Comment:
This magic number makes me feel filthy. Could we reference it using the R class perhaps?

Reply:
@colintheshots I have no idea what that magic number is :( I pulled the implementation out from Snackbar so we wouldn't have to reference the rootView ourselves.

Reply:
Quick search of the docs and it looks like `R.id.content` I'll test it now